pkg/cfaws/assumer_aws_sso.go

```diff
@@ -166,10 +175,12 @@ func SSODeviceCodeFlowFromStartUrl(ctx context.Context, cfg aws.Config, startUrl
 	}
 	// trigger OIDC login. open browser to login. close tab once login is done. press enter to continue
 	url := aws.ToString(deviceAuth.VerificationUriComplete)
-	fmt.Fprintf(color.Error, "If browser is not opened automatically, please open link:\n%v\n", url)
 
-	//check if sso browser path is set
+	if !shouldSilentLogs {
```

Comment:
Do we need this? If we're printing to `color.Error` we're printing to stderr rather than stdout, so it shouldn't cause any issues?

Reply:
Doesn't work. Yeah that was my understanding as well. I even tried to create a custom log and send to stderr but didn't work.